templates/takeovers/_ai_webinar.html

```diff
@@ -1,10 +1,10 @@
-<section class="p-strip is-deep p-takeover--ai-webinar u-hide js-primary-takeover">
+<section class="p-strip is-deep p-takeover--ai-webinar u-hide takeover">
```

Comment:
Could you please prefix the`takeover` class with js to indicate it adds functionally and not styling.